components/applications-service/pkg/storage/postgres/db.go

```diff
@@ -129,6 +129,7 @@ func (s *supervisor) NeedUpdate() bool {
 // serviceGroup struct is the representation of the service_group table inside the db
 type serviceGroup struct {
 	ID           int32     `db:"id"`
+	HashId       string    `db:"hash_id"`
```

Comment:
this is leftover from something else, but should look at if I can blow away any of these struct types